modules/dynamic-plugins/con-preinstalled-dynamic-plugins.adoc

```diff
@@ -2,9 +2,17 @@
 
 = Preinstalled dynamic plugins
 
-{product} is preinstalled with a selection of dynamic plugins. The dynamic plugins that require custom configuration are disabled by default.
+{product} is preinstalled with a selection of dynamic plugins. For a complete list of dynamic plugins that are preinstalled in this release of {product-short}, see xref:rhdh-supported-plugins[Preinstalled dynamic plugin descriptions and details].
```

Comment:
The word "preinstalled" appears 4 times in this paragraph and its title. 

Style-wise, is that a good choice?